apps/release/release/builder.py

```
@@ -0,0 +1,428 @@
+    tag: bool = False,
+    all: bool = False,
+    force: bool = False,
+    package: Package = DEFAULT_PACKAGE,
+    creds: Optional[Credentials] = None,
+    stash: bool = False,
+) -> None:
+    from .network import fetch_pypi_releases
+    from .uploader import upload_with_retries
+
+    if all:
+        bump = dist = twine = git = tag = True
```

Ruff (0.14.14)

[error] Local variable `bump` is assigned to but never used

Remove assignment to unused variable `bump`

(F841)